plugins/foreman_templates/5.0/index.md

```diff
@@ -202,7 +202,45 @@ When there is OrgA with template some_tmpl and b_user (user only in OrgB) tries
 
 # 7. Help
 
-Please follow our [standard procedures and contacts]({{site.baseurl}}support.html).
```

Comment:
+1 this seems to be different to the title of the PR and should probably have PR of it's own.